hub-config/tasks.json

```diff
@@ -1,25 +1,224 @@
+  "rounds": [
+    {
+      "round_id_from_variable": true,
+      "round_id": "nowcast_date",
+      "model_tasks": [
         {
-            "round_id_from_variable": true,
-            "round_id": "origin_date",
-            "model_tasks": [
-                {
-                    "task_ids": {
-                        "origin_date": {
-                            "required": [],
-                            "optional": []
-                        }
-                    },
-                    "output_type": {},
-                    "target_metadata":[]
-                }
-            ],
-            "submissions_due": {
-                "start": "1900-01-01",
-                "end": "2900-12-31"
+          "task_ids": {
+            "nowcast_date": {
+              "required": [
+                "2024-09-11"
+              ],
+              "optional": null
+            },
+            "target_date": {
+              "required": null,
+              "optional": ["2024-09-11", "2024-09-04", "2024-08-28", "2024-08-21"]
+            },
+            "location": {
+              "required": null,
+              "optional": ["AL", "AK", "AZ", "AR", "CA", "CO", "CT", "DE", "DC", "FL", "GA", "HI", "ID", "IL", "IN", "IA", "KS", "KY", "LA", "ME", "MD", "MA", "MI", "MN", "MS", "MO", "MT", "NE", "NV", "NH", "NJ", "NM", "NY", "NC", "ND", "OH", "OK", "OR", "PA", "RI", "SC", "SD", "TN", "TX", "UT", "VT", "VA", "WA", "WV", "WI", "WY", "PR"]
+            },
+            "variant": {
+              "required": ["24A", "24B", "24C", "other", "recombinant"],
+              "optional": null
             }
+          },
+          "output_type": {
+            "mean": {
+              "output_type_id": {
+                "required": null,
+                "optional": [
+                  "NA"
+                ]
+              },
+              "value": {
+                "type": "double",
+                "minimum": 0,
+                "maximum": 1
+              }
+            },
+            "sample": {
+              "output_type_id_params": {
+                "is_required": false,
+                "type": "character",
+                "max_length": 15,
+                "min_samples_per_task": 1,
+                "max_samples_per_task": 500
+              },
+              "value": {
+                "type": "double",
+                "minimum": 0,
+                "maximum": 1
+              }
+            }
+          },
+          "target_metadata": [
+            {
+              "target_id": "variant prop",
+              "target_name": "Weekly nowcasted variant proportions",
+              "target_units": "proportion",
+              "target_keys": null,
+              "target_type": "compositional",
+              "is_step_ahead": true,
+              "time_unit": "week"
+            }
+          ]
+        }
+      ],
+      "submissions_due": {
+        "relative_to": "nowcast_date",
+        "start": -7,
```

Comment:
I think we should set this to be -2 because this is saying we will accept submissions seven days prior to the nowcast_date but the task for a Wednesday nowcast date will never be defined earlier than 2 days prior (the Monday).